admin/menu.php

```diff
@@ -42,6 +42,29 @@ function extensions() {
 }
 
 /**
+ * Highlights the 'Backups' submenu item when on the Extentions page
+ *
+ * @param string $parent_file The filename of the parent menu item
+ * @return string $parent_file The filename of the parent menu item
+ */
+function highlight_submenu( $parent_file ) {
+
+	global $submenu_file;
+
+	$screen = get_current_screen();
+
+	if ( 'tools_page_' . HMBKP_PLUGIN_SLUG . '_extensions' === $screen->id ) {
+
+		// Set the main plugin page to be the active submenu page
+		$submenu_file = HMBKP_PLUGIN_SLUG;
+
+	}
+
+	return $parent_file;
+}
+add_filter( 'parent_file', 'HM\BackUpWordPress\highlight_submenu' );
```

Comment:
Thanks @shadyvb, I did look at using `submenu_file` the first time but couldn't get it to work. Just had another look and got it working this time :) thanks!